tools/cldr-apps/src/main/webapp/js/new/cldrText.js

```diff
@@ -286,7 +286,7 @@ const cldrText = (function () {
       "${count} old winning votes were automatically imported",
     "v-title_desc":
       "This area shows the date before which votes are considered “old”.",
-    special_account: "Settings",
+    special_account: "Account Settings",
     special_admin: "Admin Panel",
```

Comment:
"Account Settings" is now the top-level description for both "My Account, Settings" and "List ... Users"